services/app-api/src/emailer/emails/sendQuestionStateEmail.test.ts

```diff
@@ -0,0 +1,320 @@
+    modifiedWitholdAgreements: undefined,
+    modifiedStateDirectedPayments: undefined,
+    modifiedPassThroughPayments: undefined,
+    modifiedPaymentsForMentalDiseaseInstitutions: undefined,
+    modifiedMedicalLossRatioStandards: undefined,
+    modifiedOtherFinancialPaymentIncentive: undefined,
+    modifiedEnrollmentProcess: undefined,
+    modifiedGrevienceAndAppeal: undefined,
+    modifiedNetworkAdequacyStandards: undefined,
+    modifiedLengthOfContract: undefined,
+    modifiedNonRiskPaymentArrangements: undefined,
+    statutoryRegulatoryAttestation: undefined,
+    statutoryRegulatoryAttestationDescription: undefined,
+    stateContacts: [
+        {
+            name: 'test1',
+            titleRole: 'Foo1',
+            email: 'test1@example.com',
+        },
+        {
+            name: 'test2',
+            titleRole: 'Foo2',
+            email: 'test2@example.com',
+        },
+    ],
+}
+const dateAsked = new Date('01/01/2024')
+test('to addresses list includes submitter emails', async () => {
+    const sub = mockContractRev()
+    const defaultStatePrograms = mockMNState().programs
+    const template = await sendQuestionStateEmail(
+        sub,
+        defaultSubmitters,
+        cmsUser,
+        testEmailConfig(),
+        defaultStatePrograms,
+        dateAsked
+    )
+
+    if (template instanceof Error) {
+        throw template
+    }
+
+    expect(template).toEqual(
+        expect.objectContaining({
+            toAddresses: expect.arrayContaining(defaultSubmitters),
+        })
+    )
+})
+
+test('to addresses list includes all state contacts on submission', async () => {
+    const sub: ContractRevisionWithRatesType = {
+        ...mockContractRev({ formData }),
+    }
+    const defaultStatePrograms = mockMNState().programs
+    const template = await sendQuestionStateEmail(
+        sub,
+        defaultSubmitters,
+        cmsUser,
+        testEmailConfig(),
+        defaultStatePrograms,
+        dateAsked
+    )
+
+    if (template instanceof Error) {
+        console.error(template)
+        return
+    }
+
+    sub.formData.stateContacts.forEach((contact) => {
+        expect(template).toEqual(
+            expect.objectContaining({
+                toAddresses: expect.arrayContaining([contact.email]),
+            })
+        )
+    })
+})
+
+test('to addresses list does not include duplicate state receiver emails on submission', async () => {
+    const formDataWithDuplicateStateContacts = {
+        ...formData,
+        stateContacts: [
+            {
+                name: 'test1',
+                titleRole: 'Foo1',
+                email: 'test1@example.com',
+            },
+            {
+                name: 'test1',
+                titleRole: 'Foo1',
+                email: 'test1@example.com',
+            },
+        ],
+    }
+    const sub: ContractRevisionWithRatesType = {
+        ...mockContractRev({ formData: formDataWithDuplicateStateContacts }),
```

Comment:
what's up with expanding mockContractRev here instead of just using it by itself?